src/SFA.DAS.Data.Application.UnitTests/Gateways/RoatpGatewayTests/RoatpGatewayTestsBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Data.Application.Gateways;
+using SFA.Roatp.Api.Client;
+using SFA.Roatp.Api.Types;
+
+namespace SFA.DAS.Data.Application.UnitTests.Gateways.RoatpGatewayTests
+{
+    public abstract class RoatpGatewayTestsBase
+    {
+        protected RoatpGateway RoatpGateway;
+        protected Mock<IRoatpClient> RoatpClient;
+        protected IEnumerable<Roatp.Api.Types.Provider> AllProviders;
+
+        protected readonly string ValidUkPrn = "10007315";
+
+        protected Roatp.Api.Types.Provider ExpectedProvider = new Roatp.Api.Types.Provider
```

Comment:
You could use your builder for this (or just return an empty object as you should be asserting it's the same object).